src/utils/getHrefForVariant.ts

```diff
@@ -0,0 +1,10 @@
+type Args = {
+	productSlug: string;
+	variantId: string;
+};
+
+export function getHrefForVariant({ productSlug, variantId }: Args): string {
```

Comment:
We could make the `variantId` optional and then return the URL to the product page